lib/epicenter_web/templates/user_reset_password/new.html.heex

```diff
@@ -0,0 +1 @@
+
```

Comment:
Blank file?

Reply:
the original slim file seems commented out https://github.com/RatioPBC/epi-viewpoint/blob/main/lib/epicenter_web/templates/user_reset_password/new.html.slim

will remove this heex file